Addons/tsgExoticQuadrature.hpp

```diff
@@ -178,29 +193,62 @@ inline TasGrid::CustomTabulated getExoticQuadrature(const int n, const double sh
                                     std::move(weights_cache), description);
 }
 
-// Basic function overloads.
+/*
+ * \ingroup TasmanianAddonsExoticQuad
+ * \brief Constructs a CustomTabulated object containing the exotic quadrature nodes and quadrature weights for a given level \b n,
+ * scalar \b shift, a reference TasmanianSparseGrid \b grid containing loaded values of the weight function, a string \b description,
+ * and an optional bool \b is_symmetric that should be set to true if the weight function is symmetric.
+ *
+ * It is assumed that grid is a sparse grid interpolant/surrogate to the weight function. Only the nodes in grid.getLoadedPoints(),
```

Comment:
Instead of of `sparse grid` say `one dimensional`. When people hear `grid` they usually think of 2 or more dimensions and we want to emphasize that this is 1d.

Reply:
Done.